src/client/app/effects/snippet.ts

```diff
@@ -144,6 +144,29 @@ export class SnippetEffects {
             return Observable.of(new Snippet.LoadTemplatesSuccessAction([]));
         });
 
+    @Effect()
+    updateInfo$: Observable<Action> = this.actions$
+        .ofType(Snippet.SnippetActionTypes.UPDATE_INFO)
+        .map(( { payload } ) => {
+            let { id, name, description, gist, owned } = payload;
+            let snippet: ISnippet = storage.lastOpened;
+            if (storage.snippets.contains(id)) {
+                snippet = storage.snippets.get(id);
+
+                /* check if fields are undefined or null */
+                name != null ? snippet.name = name : snippet.name = snippet.name;
+                description != null ? description = payload.description : snippet.description = snippet.description;
```

Comment:
This was a typo on my part, thanks for catching it. Fixed this and modified the ternary operators to be if statements instead, since the 'snippet.field = snippet.field' is an unnecessary no-op.